docs/pages/architecture/overview.mdx

```diff
@@ -1,254 +1,133 @@
-its certificate. Clients must always connect through a proxy for two reasons:
-
-1. Individual nodes may not always be reachable from outside a secure network.
-
-2. Proxies always record SSH sessions and keep track of active user sessions.
-
-   This makes it possible for an SSH user to see if someone else is connected to
-   a node she is about to work on.
-
-### 2: Authenticate client certificate
 
-![Client offers valid certificate](../../img/cert_ok.svg)
+![Auth service](../../img/architecture/auth.png)
 
-The proxy checks if the submitted certificate has been previously signed by the
-auth server.
+</Figure>
 
-![Client obtains new certificate](../../img/cert_invalid.svg)
+<Notice type="tip">
+The auth service uses HTTPS and accepts client certificates for authentication.
+You don't have to deploy it on private net, but it's a good practice to restrict network access to it's port
+for defense in depth.
+</Notice>
 
-If there was no certificate previously offered (first time log in) or if the certificate has expired, the proxy denies the connection and asks the client to
-login interactively using a password and a 2nd factor if enabled.
+Here are some key facts about the auth service:
 
-Teleport supports
-[Google Authenticator](https://support.google.com/accounts/answer/1066447?hl=en),
-[Authy](https://www.authy.com/), or another
-[TOTP](https://en.wikipedia.org/wiki/Time-based_One-time_Password_algorithm)
-generator. The password + 2nd factor are submitted to a proxy via HTTPS,
-therefore it is critical for a secure configuration of Teleport to install a
-proper HTTPS certificate on a proxy.
+- It is the only service that has to be connected to the backend for audit and state storage.
+All other services are stateless and always interact with auth service GRPC API, never
+directly with the backend.
+- This is why you can safely limit access to the backend to the auth service only.
+- You can run multiple auth services in the cluster for high availability.
 
-<Admonition
-  type="warning"
-  title="Warning"
->
-  Do not use self-signed SSL/HTTPS certificates in production!
-</Admonition>
-
-If the credentials are correct, the auth server generates and signs a new
-certificate and returns it to the client via the proxy. The client stores this certificate
-and will use it for subsequent logins. The certificate will automatically expire after
-12 hours by default. This [TTL](https://en.wikipedia.org/wiki/Time_to_live) can be [configured](../setup/reference/cli.mdx#tctl-users-add)
-to another value by the cluster administrator.
-
-### 3: Lookup Node
-
-![Node lookup](../../img/node_lookup.svg)
+### Teleport Proxy Service
 
-At this step, the proxy tries to locate the requested node in a cluster. There
-are three lookup mechanisms a proxy uses to find the node's IP address:
+The proxy service allows access to cluster resources from the outside.
+It is the only service that has to be available from the public network.
```

Comment:
Maybe find a way to explain that "public network" here doesn't have to mean the public internet, but can be a user-facing internal network instead?